styles/src/app.styl

```diff
@@ -1,6 +1,13 @@
-html, body, #root, .app, .carouselWrapper, .react-swipeable-view-container, .slider-wrapper, .slider-wrapper .slider.animated
+html, body, #root, .app,  .slider-wrapper, .slider-wrapper .slider.animated
 	width: 100%
 	height: 100%
 	padding: 0
 	margin: 0
 
+.carouselWrapper, .react-swipeable-view-container
+	width: 100%
+	height: 100%
+
+.react-swipeable-view-container > div
+	position relative
```

Comment:
you'll need to decide sooner or later on one format that you're using, and stick with it.